tensorboard/webapp/metrics/store/metrics_store_internal_utils_test.ts

```diff
@@ -951,71 +946,63 @@ describe('metrics store utils', () => {
   });
 
   describe('getNextImageCardStepIndexFromSingleSelection', () => {
-    const cardId = 'test card Id';
-    const steps = [10, 20, 30, 40];
-
-    it(`updates nextStepIndex to matched selected time`, () => {
-      const nextStartStep = 20;
+    it(`returns step index to matched selected time`, () => {
       const nextStepIndex = getNextImageCardStepIndexFromSingleSelection(
-        cardId,
-        nextStartStep,
-        steps
+        20,
+        [10, 20, 30, 40]
       );
 
       expect(nextStepIndex).toEqual(1);
     });
 
-    it(`does not update nextStepIndex on selected step with no image`, () => {
-      const nextStartStep = 15;
+    it(`does not return step Index on selected step with no image`, () => {
       const nextStepIndex = getNextImageCardStepIndexFromSingleSelection(
-        cardId,
-        nextStartStep,
-        steps
+        15,
+        [10, 20, 30, 40]
       );
 
       expect(nextStepIndex).toEqual(null);
     });
 
-    it('updates nextStepIndex to smaller closest stepIndexIndex when they are close enough', () => {
-      const nextStartStep = 11;
+    it('returns step index to smaller closest stepIndex when they are close enough', () => {
       const nextStepIndex = getNextImageCardStepIndexFromSingleSelection(
-        cardId,
-        nextStartStep,
-        steps
+        11,
+        [10, 20, 30, 40]
       );
 
       expect(nextStepIndex).toEqual(0);
     });
 
-    it('dose not update nextStepIndex when selected step is not close to any step values', () => {
-      const nextStartStep = 12;
+    it('does not return step Index when selected step is not close to any step values', () => {
       const nextStepIndex = getNextImageCardStepIndexFromSingleSelection(
-        cardId,
-        nextStartStep,
-        steps
+        12,
+        [10, 20, 30, 40]
       );
 
       expect(nextStepIndex).toEqual(null);
     });
 
-    it('updates nextStepIndex to larger closest stepIndex when they are close enough', () => {
-      const nextStartStep = 19;
+    it('returns step index to larger closest stepIndex when they are close enough', () => {
       const nextStepIndex = getNextImageCardStepIndexFromSingleSelection(
-        cardId,
-        nextStartStep,
-        steps
+        19,
+        [10, 20, 30, 40]
       );
 
       expect(nextStepIndex).toEqual(1);
     });
 
-    it('dose not update nextStepIndex when there is only one unmatched step', () => {
-      const nextStartStep = 15;
-      const unmatchedSteps = [10];
+    it('does not return step Index when there is only one unmatched step', () => {
+      const nextStepIndex = getNextImageCardStepIndexFromSingleSelection(15, [
+        10,
+      ]);
+
+      expect(nextStepIndex).toEqual(null);
+    });
+
+    it('does not return step Index when there is no selected step', () => {
```

Comment:
"when there is no selected step" => "when there are no steps"? Looks like selected step is 15 but there are no steps